docs/source/using/addons.adoc

```diff
@@ -26,13 +26,15 @@ The following example shows the contents of an add-on file, including the name a
 # Name: anyuid                                                                          // <1>
 # Description: Allows authenticated users to run images under a non pre-allocated UID   // <2>
 # Required-vars: ACME_TOKEN                                                             // <3>
+# OpenShift-Version: v3.6.0+                                                            // <4>
 
-oc adm policy add-scc-to-group anyuid system:authenticated                              // <4>
+oc adm policy add-scc-to-group anyuid system:authenticated                              // <5>
 ----
 <1> (Required) Name of the add-on.
 <2> (Required) Description of the add-on.
 <3> (Optional) Comma separated list of required interpolation variables. See xref:../using/addons.adoc#addon-variable-interpolation[Variable Interpolation]
-<4> Actual add-on command. In this case, the command executes the *oc* binary.
+<4> (Optional) If provided then it will compare with runing OpenShift version in minishift and throw an error in case add-on require an higher version.
```

Comment:
And what is the syntax? Exact matches only or is there a range semantic as well. How does it look like? I think there is more needed than just this short reference. There is a dedicated section needed for this as well.